src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.h

```diff
@@ -82,6 +82,8 @@ class OPENMS_DLLAPI FeatureFinderIdentificationAlgorithm :
   PeakMap& getMSData() { return ms_data_; }
```

Comment:
Should the definition of the getters and setter be moved to the source file?

Reply:
yes please